doc/releases/1.16.1dev.rst

```diff
@@ -17,6 +17,7 @@ Dependency Changes
 
 Functionality/Performance Improvements and Additions
 ----------------------------------------------------
+- Added support for data coaddition with the UVES_popler GUI tool.
```

Comment:
I would reword this:  "Adjusted spec1d datamodel to enable use with UVES_popler GUI tool"